src/router/index.js

```diff
@@ -10,6 +10,8 @@ const RepoTracker = () =>
   import(/* webpackChunkName: "repoTracker" */ "@/views/RepoTracker.vue");
 const LoginPage = () =>
   import(/* webpackChunkName: "loginPage" */ "@/views/LoginPage.vue");
+const NewUser = () =>
+  import(/* webpackChunkName: "loginPage" */ "@/views/NewUser.vue");
```

Comment:
this needs to be under its own chunk.
```suggestion
  import(/* webpackChunkName: "newUser" */ "@/views/NewUser.vue");
```